File: XBOOK/XBOOK.Data/ViewModels/PaymentViewModel.cs
using System;

namespace XBOOK.Data.ViewModels
{
    public class PaymentViewModel
    {
        public long Id { get; set; }
        public long InvoiceId { get; set; }
        public DateTime PayDate { get; set; }
        public int PayTypeID { get; set; }
        public string PayType { get; set; }
        public string ReceiptNumber { get; set; }
        public decimal Amount { get; set; }
        public string Note { get; set; }
    }
}

File: XBOOK/XBOOK.Web/ClientApp/src/app/pages/_shared/models/invoice/payment-view.model.ts
import { BaseModel } from '../../../../shared/model/base.model';

export class PaymentView extends BaseModel {
  invoiceId: number;
  productId: number;
  payDate: string;
  payTypeId: number;
  payType: string;
  receiptNumber: string;
  amount: number;
  note: string;
  payName: string;
}
